pages/reset-pass.tsx

```diff
@@ -0,0 +1,31 @@
+import Button from "@/components/shared/buttons/Button/Button";
+import Input from "@/components/shared/forms/Input/Input";
+import Layout from "@/components/shared/layout/Layout";
+import Heading from "@/components/shared/typography/Heading/Heading";
+import Link from "next/dist/client/link";
+
+const RestPass = () => {
```

Comment:
NIT: haven't you meant `ResetPass` ? Just in case, It's good practice to avoid shortcuts and write the full names